app-libs/stf/src/helpers.rs

```diff
@@ -119,21 +121,19 @@ pub fn set_block_number(block_number: u32) {
 
 // Litentry
 pub fn aes_encrypt_default(key: &UserShieldingKeyType, data: &[u8]) -> AesOutput {
-	// it requires "std" but it shouldn't be a problem
-	let nonce = Aes256Gcm::generate_nonce(&mut OsRng);
-	aes_encrypt(key, data, b"", nonce.into())
+	aes_encrypt(key, data)
 }
 
-pub fn aes_encrypt(
-	key: &UserShieldingKeyType,
-	data: &[u8],
-	aad: &[u8],
-	nonce: [u8; USER_SHIELDING_KEY_NONCE_LEN],
-) -> AesOutput {
-	let cipher = Aes256Gcm::new(key.into());
-	let payload = Payload { msg: data, aad };
-	let ciphertext = cipher.encrypt(&nonce.into(), payload).unwrap();
-	AesOutput { ciphertext, aad: aad.to_vec(), nonce }
+pub fn aes_encrypt(key: &UserShieldingKeyType, data: &[u8]) -> AesOutput {
```

Comment:
I'd say we either:
- keep both `aes_encrypt_default` and `aes_encrypt` with the latter in a more generic form
- or consolidate them into one fn like  `aes_encrypt_default` where aad and nonce are not configurable.

Reply:
Done. Consolidate them together.